### ISSUE TITLE
Echo_Upscaleloader model issue

### ISSUE THREAD

**Issue by chenxiuqi2** [smthemex/ComfyUI_EchoMimic#70]
When using Echo_Upscaleloader, the model didn't download automatically. I manually downloaded detection_Resnet50_Final.pth and RealESRGAN_x2plus.pth, but I'm not sure which folder to put them in. Can you help me out?

**Comment by smthemex** [comment]
Yes, I forgot to include the automatic download code for these two models
